realsense2_camera/src/base_realsense_node.cpp

```diff
@@ -825,8 +825,17 @@ void BaseRealSenseNode::setupPublishers()
             _image_publishers[stream] = image_transport.advertise(image_raw.str(), 1);
             _info_publisher[stream] = _node_handle.advertise<sensor_msgs::CameraInfo>(camera_info.str(), 1);
 
+            // Setup diagnostics
             const std::string diagnostic_name("/" + _namespace + "/" + image_raw.str());
-            _updater[stream] = new marble::DiagnosticUpdater(diagnostic_name, _node_handle);
+
+            std::string display_name = "";
```

Comment:
Could we make the display name a ROS parameter instead? Rather than assuming a left and right.

Reply:
Yup, made and tested the change